src/ScottPlot/Validate.cs

```diff
@@ -29,6 +29,22 @@ public static void AssertAllReal(string label, double[] values)
                     throw new ArgumentException($"{label} index {i} is invalid ({values[i]})");
         }
 
+        /// <summary>
+        /// Throw an exception if the array is null or contains NaN or infinity
+        /// </summary>
+        public static void AssertAllReal<T>(string label, T[] values)
+        {
+            if (values is null)
```

Comment:
label also can be null or empty,
Maybe add some check for this?

Reply:
Good catch! I can add something simple like:

```cs
label = string.IsNullOrWhiteSpace(label) ? "[unknown variable]" : label;
```